README.md

```diff
@@ -15,18 +14,24 @@ The implemented hashing functions are the following.
    - Heavily based on this java implementation: [@romus/sha](https://github.com/romus/sha)
    - Tested with [Nist test vectors](https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Algorithm-Validation-Program/documents/sha3/sha-3bittestvectors.zip)
 
+## Supported platforms
+| Platform | Supported |
+| :------- | :-------: |
+| `common` | Yes | 
+| `jvm` | Yes | 
+| `js` | No |
+| `native` | No |
```

Comment:
bit puzzled about this - my understanding is when 'common' Yes - then you do not need platform specific code. Also @tobiasheine recently told me he is using it for iOS - so 'native' no would be wrong. Would still merge (as it is then more over-delivering than over-promising) - but curious and something that catched my eye when reviewing

Reply:
I've updated the README to make it more clear. 

Unfortunately, two modules are not usable inside the `native` and `js` platforms. Those modules are: 

- `keccak`
   This is due to the `String.parseInt` extension function, which is not available in Kotlin, used inside the `convertFrom64ToLittleEndian` method. 

- `khash-extensions` 
   Due to the `Char.digit` method used inside the `Char.getNibbleValue` method that is itself used inside the `String.hexToByteArray` extension function. This could be solved by merging [KHex/GH-1](https://github.com/komputing/KHex/pull/1) and using the extension functions defined there instead of repeating the code

Reply:
Regarding `String.parseInt`, couldn't you use [String.toInt](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-int.html)?

Reply:
Good point, pushed the change right now